src/Groups/group_characters.jl

```diff
@@ -275,8 +286,49 @@ end
 #
 # admissible names of library character tables
 
-function all_character_table_names()
-    K = GAP.Globals.CallFuncList(GAP.Globals.AllCharacterTableNames, GAP.GapObj([]))
+"""
+    is_duplicate_table(tbl::GAPGroupCharacterTable)
+
+Return whether `tbl` is a table from the character table library
+that was constructed from another library character table by permuting rows
+and columns.
+
+One application of this function is to restrict the search with
+`all_character_table_names` to only one library character table for each
+class of permutation equivalent tables.
+"""
+@gapattribute is_duplicate_table(tbl::GAPGroupCharacterTable) = GAP.Globals.IsDuplicateTable(G.GAPTable)::Bool
+
+"""
+    all_character_table_names(L...)
+
+Return an array of strings that contains all those names of character tables
+in the character table library that satisfy the conditions in the array `L`.
+"""
+function all_character_table_names(L...; ordered_by = nothing)
+    @assert CheckValidType(L)[1] "Wrong type inserted"
+
+    L1 = Any[]
+    hadfunc = false
+    for i in 1:length(L)
+      if typeof(L[i]) <: Function
+        if hadfunc
+          push!(L1, true)
+        end
+        push!(L1, find_index_function(L[i],false)[2])
+        hadfunc = true
+      else
+        push!(L1, GAP.julia_to_gap(L[i]))
+        hadfunc = false
+      end
+    end
+
+    if ordered_by isa Function
+      K = GAP.call_gap_func(GAP.Globals.AllCharacterTableNames, L1...;
+            OrderedBy = find_index_function(ordered_by, false)[2])
+    else
+      K = GAP.call_gap_func(GAP.Globals.AllCharacterTableNames, L1...)
```

Comment:
Doesn't this work, too?
```suggestion
      K = GAP.Globals.AllCharacterTableNames(L1...)
```

Reply:
Yes.